plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -909,6 +915,24 @@ private void populateSplits(File war) throws IOException {
         }
         throw new IOException("no jenkins-core-*.jar found in " + war);
     }
+
+    private List<String> removeSplitsBasedOnJDK(List<String> splits, VersionNumber jdkVersion) {
+        List<String> filterSplits = new LinkedList();
+        for (String split : splits) {
+            String[] tokens = split.trim().split("\\s+");
+            if (tokens.length == 4 ) { // We have a jdk field in the splits file
+                if (jdkVersion.isNewerThan(new VersionNumber(tokens[3]))) {
+                    filterSplits.add(split);
+                } else {
+                    System.out.println("Not adding " + split + "as split because jdk specified " + tokens[3] + " is newer than running jdk " + jdkVersion);
```

Comment:
```suggestion
                    System.out.println("Not adding " + split + " as split because jdk specified " + tokens[3] + " is newer than running jdk " + jdkVersion);
```